src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -329,68 +299,28 @@ Unix support is disabled by default. See https://aka.ms/systemdrawingnonwindows
              Link="Common\Interop\Windows\User32\Interop.WindowFromDC.cs" />
     <Compile Include="$(CommonPath)Interop\Windows\Interop.HRESULT.cs"
              Link="Common\Interop\Windows\Interop.HRESULT.cs" />
+    <EmbeddedResource Include="Resources\System\Drawing\DefaultComponent.bmp">
+      <LogicalName>System.Drawing.DefaultComponent.bmp</LogicalName>
+    </EmbeddedResource>
+    <EmbeddedResource Include="Resources\System\Drawing\Printing\PrintDocument.bmp" />
+    <EmbeddedResource Include="Resources\System\Drawing\ShieldIcon.ico">
+      <LogicalName>System.Drawing.ShieldIcon.ico</LogicalName>
+    </EmbeddedResource>
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsWindows)' == 'true' and '$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
```

Comment:
This move it to the `targetswindows` block as well, .NET Framework build will never set that to true as it doesn't have a windows suffix on the tfm.

Reply:
Done.

Reply:
I changed the condition to `'$(TargetPlatformIdentifier)' != ''`, the same with the source files. I prefer to keep the assembly and project references in a separate group for clarity and to minimize layout changes.

Reply:
It doesn't work, CI failed. Haven't actually understood what you asked me to do. 😵‍💫